front/app/components/InitiativeForm/index.tsx

```diff
@@ -1,5 +1,4 @@
 import * as React from 'react';
-import { get } from 'lodash-es';
 import { stripHtmlTags, isNilOrError } from 'utils/helperUtils';
```

Comment:
For this component, I first refactored it from a class component to a functional one. There were also a number of bugs (on production) which I fixed. I don't really like how this component works (on load it created a new proposal, then every time anything is changed, it sends a patch request to the BE to save it). On the final "Publish" button click, it sends a request which changes the status from draft to published.

Ideally the component shouldn't work this way, and we should be using JSON schemas like we have for ideation. That's out of scope though for this PR, so I just tried to clean things up a bit.